.travis.yml

```diff
@@ -7,20 +7,19 @@ python:
   - "3.6"
 
 install:
-  - pip install --upgrade ansible ansible-lint
+  - pip install --upgrade ansible
   - pip install --upgrade sphinx sphinx_rtd_theme
 
 before_script:
   - export ANSIBLE_VAULT_PASSWORD_FILE=".testenv/vault-env"
   - ERRORS=0
   - git submodule update --init
   - ansible --version
-  - ansible-lint --version
   - mkdir ansible-guide
+  - ansible-galaxy install -r requirements.yml
 
 script:
   - ansible-playbook site.yml --syntax-check || let ERRORS+=1
-  - ansible-lint -R -r .testenv/lint-rules/ site.yml || let ERRORS+=1
```

Comment:
Why does Travis no longer perform any custom linting checks?

Reply:
It's not needed. The Ansible Guide shouldn't fail because a role need to be lintet. The CI check on the specific role has to fail and the role has to be fixed.